locales/en-US/app.ftl

```diff
@@ -85,6 +84,9 @@ emailOptInDialogErrorTitle = Oh no!
 latestUpdatesTitle = Latest updates
 showMoreNewsTitle = Show Past News
 
+[[featuredExperiment]]
+moreDetail = More Detail
```

Comment:
Not l10n related, but this doesn't sound right (details)?

Reply:
I think works both ways but whichever sounds most natural should go in